plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/editor/JavaCodeAssistProcessor.java

```diff
@@ -137,11 +145,12 @@ public static String insertStyle(final JavaResources javaResources, final String
         }
     }
 
-    public static ImageResource getImage(final String image) {
-        if (image == null) {
-            return null;
+    public static Icon getIcon(final String image) {
+        if (svgs.containsKey(image)) {
+            return new Icon("", svgs.get(image));
         }
-        return images.get(image);
+
+        return new Icon("", images.get(image));
```

Comment:
I think we can simplify it
return svgs.containsKey(image) ? new Icon("", svgs.get(image)) : new Icon("", images.get(image));